src/content/docs/en/guides/assets.mdx

```diff
@@ -65,23 +96,9 @@ const blogCollection = defineCollection({
     image: asset({ width: 1120, height: 1120 }),
   }),
 });
-
 ```
 
-Additionally, assets referred this way will be transformed automatically to the same shape as if the image was imported, so they're ready to be used optimally.
-
-## New signature for images imports
-
-Previously, importing an image would return a simple `string` with the path of the image. With the new `image` features enabled, imported image assets now match the following signature:
-
-```ts
-interface ImageAsset {
-  src: string;
-  width: number;
-  height: number;
-  format: string;
-}
-```
+Assets referred this way will be transformed automatically to the same shape as if the image was imported, so they're ready to be used optimally.
```

Comment:
> so they're ready to be used optimally.

I'd avoid the term optimally, as it might trick users into thinking the image is optimized. You still have to pass the image through `getImage` or the Image component to optimize it